python/instrumentation/openinference-instrumentation-litellm/tests/test_instrumentor.py

```diff
@@ -0,0 +1,319 @@
+import json
+from unittest.mock import patch
+
+import litellm
+import pytest
+from litellm.llms.openai import OpenAIChatCompletion
+from openinference.instrumentation.litellm import LiteLLMInstrumentor
+from openinference.semconv.trace import EmbeddingAttributes, ImageAttributes, SpanAttributes
+from opentelemetry.sdk.resources import Resource
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import SimpleSpanProcessor
+from opentelemetry.sdk.trace.export.in_memory_span_exporter import InMemorySpanExporter
+
+
+@pytest.fixture(scope="class")
+def in_memory_span_exporter() -> InMemorySpanExporter:
+    return InMemorySpanExporter()
+
+
+@pytest.fixture(scope="class")
+def tracer_provider(in_memory_span_exporter: InMemorySpanExporter) -> TracerProvider:
+    resource = Resource(attributes={})
+    tracer_provider = TracerProvider(resource=resource)
+    tracer_provider.add_span_processor(SimpleSpanProcessor(in_memory_span_exporter))
+    return tracer_provider
+
+
+@pytest.fixture(scope="class")
+def instrumentor(tracer_provider: TracerProvider) -> LiteLLMInstrumentor:
+    instrumentor = LiteLLMInstrumentor(tracer_provider=tracer_provider)
+    instrumentor.instrument()
+    yield instrumentor
```

Comment:
FWIW, since the yielded `instrumentor` is not used in any of the tests, we can simply `yield` here

Reply:
Given that this is two lines of procedure that is fairly standard for instrumentation, I'd be tempted to suggest to not make this a fixture at all and simply include them in all of the tests

Reply:
Done-- brought the 2 lines of code into each test functions and removed instrumentor fixture